react/IconClose.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import Icon from './components/Icon'
+import { proptypes } from './components/propTypes'
+import { getType } from './components/helpers'
+
+const IconClose = ({ type, ...props }: EnhancedIconProps) => {
+  const typeModifier = getType(type)
+  return <Icon id={`sti-close${typeModifier}`} {...props} />
+}
+
+IconClose.propTypes = proptypes
```

Comment:
Whats the point of having propTypes since you are already using typescript?

Reply:
This lib is consumed by a lot of projects that are not typescript based, so prop-types is useful for them. Typescript, also, does not provide runtime check, right?

Reply:
If u say so. It doesn't but it doesn't really matter.

Reply:
If doesn't matter that much, I'm removing `prop-types`. Other typescript-based projects do not use it, so i'll follow that pattern as well. Thanks for the tip.